src/app.ts

```diff
@@ -25,7 +25,7 @@ app.disable("etag");
 
 app.use(rateLimiter);
 
-app.use(cors());
+app.use(custom_cors);
```

Comment:
rename to follow camelcase format (customCors)
```suggestion
app.use(customCors);
```